metrics_test.go

```diff
@@ -606,10 +606,12 @@ func TestResourceByDescriptor(t *testing.T) {
 						},
 						LabelValues: []metricdata.LabelValue{
 							{
-								Value: "v11",
+								Present: true,
```

Comment:
Could you please add a unit test for `Present: false`?

Reply:
Done.